### PR TITLE
[9.0][event_contact] Migration to v9

### PR DESCRIPTION
**Pull request by mikevhe18**

### CODE REVIEW

**Comment by rafaelbn** [comment]
Hi @mikevhe18 , please review runbot. Thanks!!

**Comment by mikevhe18** [comment]
@rafaelbn It's done, i've made a changes. Would you kindly review this PR again ?

File: event_contact/tests/__init__.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-# © 2016 Antiun Ingeniería S.L. - Jairo Llopis
```
Comment (yajo) [inline event_contact/tests/__init__.py]: Please don't remove others' copyright, just add yours below if you want. :wink:

BTW you can change this one to `Copyright 2016 Jairo Llopis <jairo.llopis@tecnativa.com>`

File: event_contact/tests/test_event.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
-# © 2016 Antiun Ingeniería S.L. - Jairo Llopis
```
Comment (yajo) [inline event_contact/tests/test_event.py]: Same.

File: event_contact/views/event_event_view.xml [hunk 3]
```diff
@@ -2,14 +2,14 @@
 <openerp>
     <data>
 
-        <record model="ir.ui.view" id="event_contacts_form_view">
+        <record model="ir.ui.view" id="event_event_form_view">
```
Comment (yajo) [inline event_contact/views/event_event_view.xml]: Better not change XMLID if not strictly required. Complicates things for migrations.

File: event_contact/views/event_event_view.xml [hunk 4]
```diff
@@ -2,14 +2,14 @@
 <openerp>
     <data>
```
Comment (yajo) [inline event_contact/views/event_event_view.xml]: `<odoo>`

File: event_contact/views/event_type_view.xml [hunk 5]
```diff
@@ -1,20 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
-<!-- © 2016 Antiun Ingeniería S.L. - Jairo Llopis
-     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
-
 <openerp>
-<data>
+	<data>
```
Comment (yajo) [inline event_contact/views/event_type_view.xml]: `<odoo>`

File: event_contact/views/event_type_view.xml [hunk 6]
```diff
@@ -1,20 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
-<!-- © 2016 Antiun Ingeniería S.L. - Jairo Llopis
-     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
```
Comment (yajo) [inline event_contact/views/event_type_view.xml]: As said before...

File: event_contact/views/event_type_view.xml [hunk 7]
```diff
@@ -1,20 +1,17 @@
...
-    <field name="name">Add contacts</field>
-    <field name="model">event.type</field>
-    <field name="inherit_id" ref="event.view_event_type_form"/>
-    <field name="arch" type="xml">
-        <xpath expr="//group">
-            <field name="contact_ids" widget="many2many_tags"/>
-        </xpath>
-    </field>
-</record>
+		<record id="event_type_form_view" model="ir.ui.view">
```
Comment (yajo) [inline event_contact/views/event_type_view.xml]: Same about XMLID.

**Review by yajo — CHANGES_REQUESTED** [review]
Don't erase me, pal! :laughing: 

All will be OK once these simple changes are made :wink:

**Comment by mikevhe18** [comment]
Sorry @Yajo, it's my fault. 
I already fix it.

**Review by andhit-r — APPROVED** [review]
:+1:

**Comment by pedrobaeza** [comment]
@Yajo, please check your review now that this has been changed

File: website_event_contact/views/website_event_contact.xml [hunk 8]
```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+       
+        <template id="website_event_contact_template" inherit_id="website_event.event_description_full">
+            <xpath expr="//div[@class='col-md-4 css_noprint']" position="replace">
```
Comment (yajo) [inline website_event_contact/views/website_event_contact.xml:6]: Wrong replacement.

You should just add the little part you need (the contacts one, right after Organizer). You are instead overwriting many things you don't need to. This means problems for sure.

**Review by yajo — CHANGES_REQUESTED** [review]

File: website_event_contact/views/website_event_contact.xml (hunk 8 above)
Comment (rafaelbn) [reply website_event_contact/views/website_event_contact.xml:6]: @mikevhe18 could you review this comment please? Thanks!

**Review by rafaelbn — APPROVED** [review]
Functionally tested in runbot 👍 

Please @mikevhe18 review https://github.com/OCA/event/pull/61#pullrequestreview-23932398 in order to merge! thanks

**Pull request closed without merging**